src/CoreWCF.Templates/src/templates/CoreWCFService/CoreWCFService.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <!--#if(nullableEnabled)-->
```

Comment:
We have support for targeting .NET Framework, but this looks hard coded to .net6.0. Should there be an if block around this?

Reply:
I double checked this is correctly valued dynamically using the optional `--framework` arg when using dotnet CLI. Using the VS GUI you may also use the drop down menu allowing to choose the target framework. https://github.com/CoreWCF/CoreWCF/pull/590#discussion_r861507711

Reply:
Sounds like it's just a placeholder then. That's really non-intuitive. Thanks for checking.

Reply:
indeed. You can have a look to the integration tests here:
https://github.com/CoreWCF/CoreWCF/blob/3f3e41801daeb787f1cfc86bfc202fcce12fd6b4/src/CoreWCF.Templates/tests/BasicTests.cs
Basically CoreWCF template is installed and then the unit test code is issuing all variations of `dotnet new corewcf`. Then the project is built and ran. Finally an HttpClient ensure it is working appropriately.